config/config.js

```diff
@@ -70,6 +70,12 @@ module.exports = function (fs, path, url, convict) {
       format: 'duration',
       env: 'SIGNIN_CODES_MAX_AGE',
     },
+    requiredSQLModes: {
+      doc: 'Comma-separated list of SQL mode flags to enforce on each connection',
+      default: '',
```

Comment:
Looks good, let's just not forget to actually set this at some point :)

Reply:
If you leave it unset, it should use the hard-coded defaults from `const REQUIRED_SQL_MODES`; perhaps the config doc here could have made that clearer.